graph/incremental_minimum_spanning_forest/info.toml

```
@@ -0,0 +1,18 @@
+title = 'Incremental Minimum Spanning Forest'
+timelimit = 5.0
```

I implemented it as well, but I think 5 seconds is tight given the constraints, so I suggest 10 seconds.
```suggestion
timelimit = 10.0
```